source/reference/indexes.txt

```diff
@@ -1,278 +1,213 @@
+about index options, see the :ref:`mongoid-indexes-api-documentation` section.
 
-.. code-block:: ruby
+Aliases and Declaring Indexes
++++++++++++++++++++++++++++++
 
-    class Person
-      include Mongoid::Document
-      field :first_name
-      field :last_name
+You can use aliased field names in index definitions. For example, the following
+example creates a field on the ``b`` field, which is an alias of the ``borough``
+field:
 
-      index({ first_name: 1, last_name: 1 }, { unique: true })
-    end
+.. literalinclude:: /includes/reference/indexes.rb
+  :language: ruby
+  :start-after: start create alias index
+  :end-before: end create alias index
 
-Indexes can be sparse:
+Create an Index on Embedded Document Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: ruby
+You can define an index on embedded document fields. The following code example
+shows how to declare an ascending index on the ``street`` field in the
+``Restaurant`` class where ``street`` is a field embedded within the ``address``
+field.
 
-    class Person
-      include Mongoid::Document
-      field :ssn
+.. literalinclude:: /includes/reference/indexes.rb
+  :language: ruby
+  :start-after: start create embedded index
+  :end-before: end create embedded index
 
-      index({ ssn: -1 }, { sparse: true })
-    end
+Create a Compound Index
+~~~~~~~~~~~~~~~~~~~~~~~
 
-For geospatial indexes, make sure the field being indexed is of type Array:
+You can define a compound index on multiple fields. The following code example
+shows how to declare a compound index that is ascending on the ``address.street``
+field and descending on the ``name`` field.
 
-.. code-block:: ruby
+.. literalinclude:: /includes/reference/indexes.rb
+  :language: ruby
+  :start-after: start create compound index
+  :end-before: end create compound index
 
-    class Person
-      include Mongoid::Document
-      field :location, type: Array
+Create a Geospatial Index
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      index({ location: "2d" }, { min: -200, max: 200 })
-    end
+You can define a 2dsphere index on fields that contain GeoJSON objects or
+coordinate pairs.
+The following example defines a 2dsphere index on a field that contains GeoJSON
+objects:
 
-Indexes can be scoped to a specific database:
+.. literalinclude:: /includes/reference/indexes.rb
+  :language: ruby
+  :start-after: start create 2dsphere index
+  :end-before: end create 2dsphere index
 
-.. code-block:: ruby
+For more information on 2dsphere indexes, see the :manual:`2dsphere </core/indexes/index-types/geospatial/2dsphere/>`
+guide in the {+mdb-server+} manual.
 
-    class Person
-      include Mongoid::Document
-      field :ssn
-      index({ ssn: 1 }, { database: "users", unique: true, background: true })
-    end
+For more information on the GeoJSON type, see the :manual:`GeoJSON Objects </reference/geojson/>`
+guide in the {+mdb-server+} manual.
```

Comment:
Oh it looks like this variable isn't in `snooty.toml`. Instead I see `server-manual = "Server manual"` so just need to update here.